content/blog/intro-to-html-css-and-savascript/index.md

```diff
@@ -0,0 +1,71 @@
+---
+{
+    title: "Introduction to HTML, CSS, and JavaScript",
+    description: 'Introduction to the underlying concepts of HTML, CSS, and JavaScript and how they work together.',
+    published: '2019-12-16T13:45:00.284Z',
+    authors: ['MDutro'],
+    tags: ['HTML', 'CSS', 'JavaScript', 'Beginner'],
+    attached: [],
+    license: 'publicdomain-zero-1'
+}
+---
+
+So you have decided to learn web development.
+
+Great! But once you start looking around for tutorials, guides, and other resources it’s easy to get overwhelmed. Web development incorporates a lot of different technologies that all work together to create the internet as we know it today. Understanding how they work together is no small task, especially when you are at the beginning of your web development journey. Which ones should you learn first? What do they do? What does it all mean???
+
+Unicorn Utterances is here to help. Let’s start at the beginning. 
```

Comment:
```suggestion
Unicorn Utterances is here to help. Let’s start at the beginning.
```

- Removed space at the end of the line (as do some of the suggestions that follow)